clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/DockerIntegrationSpec.scala

```diff
@@ -86,7 +85,11 @@ class DockerIntegrationSpec
     )
     val logStream = clioLogLines
       .map { line =>
-        if (!line.contains("DEBUG")) println(line)
```

Comment:
Printing not-debug picks up a ton of spam from elastic4s that gets logged out during recovery.

Reply:
Can you use a custom logback.xml file to fix this instead? That would be cleaner than adding a filter to the logging output.

Reply:
We've already got a custom logback conf causing the clio in Docker to log to file at debug level, which I think is what we want so when tests fail we have the info to potentially track down the problem. This block of code is tailing the log file looking for the "Server started" message (so it knows when to start the tests), printing intermittent lines so it doesn't look like the test has hung. After that, we don't print any server logging to console to try to keep the test output as de-cluttered as possible.

Alt: we could modify the custom logback conf to also have a console appender at info level, and configure testcontainers to auto-tail and print container console logs. I'd rather not do this because:
1. I think we'd then get spammed by all the elasticsearch container logs
2. We write audit logs at info level, so we'd definitely get spammed by all of those logs